carbon-design-system/carbon
# fix: fixed VO and JAWS problem in FF and Safari

Closes #13888 

Added `tabIndex={-1}` to avoid that screen readers passes throught it.

_I tested only with VO, but it should work on JAWS also._

#### Testing / Reviewing

- Open the FireFox and Safari
- Open Modal component
- Tab/Arrow in the component

You should expect the same behaivor as Chrome in both browsers
